tests/test_notifications.py

```diff
@@ -0,0 +1,39 @@
+from .base import BaseTestCase
+
+
+class TestNotifications(BaseTestCase):
+
+    SUCCESS_CODE = 200
+
+    def test_get_delete_notifications(self):
+        # get all notifications
+        res = self.nxc.get_notifications()
+        all_data = res['ocs']['data']
```

Comment:
This is interesting, the test looks like that it may end up either way - there are or aren't any notifications. I would appreciate more control - what about deleting, asserting that there are none, and then adding and rediscovering them. Does it make sense?

Reply:
Yes, you right. As far as I researched, right now there is no possibility to create actions using api. It will be possible if we have WebDAV api implemented, which will let us make files related changes, writing comments, or Federated files sharing api, which also creates activity objects when sharing with another cloud. Also this is related to notifications (#16) tests.

Noticed it in code below as well:
```
        # TODO: add more tests if WebDAV api will be implemented and there will be ability to make actions
        #  using api which creates notifications (mentions in comments)
        #  or when Federated file sharings api will be implemented (harder way)
```

Reply:
Great!